apps/frontend/src/pages/quiz/libs/components/quiz-form/quiz-form.tsx

```diff
@@ -74,6 +74,13 @@ const QuizForm: React.FC<Properties> = ({ onNext }: Properties) => {
 		);
 	}, [currentCategoryIndex, questionsByCategories]);
 
+	useEffect(() => {
+		window.scrollTo({
+			behavior: "smooth",
+			top: NumericalValue.ZERO,
```

Comment:
```suggestion
			top: TOP_SCROLL_VALUE,
```
Let's be more descriptive